app/components/InstallExtensionButtonPlain.jsx

```diff
@@ -0,0 +1,60 @@
+import { ClientOnly } from "remix-utils";
+import UAParser from "ua-parser-js";
+
+const InstallLinks = {
+  Firefox: "https://addons.mozilla.org/en-US/firefox/addon/alby/",
+  Chrome:
+    "https://chrome.google.com/webstore/detail/alby-bitcoin-lightning-wa/iokeahhehimjnekafflcihljlcjccdbe",
+  Chromium:
+    "https://chrome.google.com/webstore/detail/alby-bitcoin-lightning-wa/iokeahhehimjnekafflcihljlcjccdbe",
+  Vivaldi:
+    "https://chrome.google.com/webstore/detail/alby-bitcoin-lightning-wa/iokeahhehimjnekafflcihljlcjccdbe",
+  Brave:
+    "https://chrome.google.com/webstore/detail/alby-bitcoin-lightning-wa/iokeahhehimjnekafflcihljlcjccdbe",
+  Opera:
+    "https://chrome.google.com/webstore/detail/alby-bitcoin-lightning-wa/iokeahhehimjnekafflcihljlcjccdbe",
+};
+
+const DefaultLink = "https://github.com/getAlby/lightning-browser-extension";
+
+function Install() {
```

Comment:
this is duplicating the code from the other components and we have to update everything in two places.

can we have one component but make it configurable? e.g. `<InstallExtensionButton type="plain">` because as far as I can see only the class attribute and the icon is different?